frontend/src/app/Settings/Facility/Components/ManageDevices.tsx

```diff
@@ -1,10 +1,9 @@
 import React from "react";
-import { v4 as uuidv4 } from "uuid";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 import Button from "../../../commonComponents/Button";
 import Dropdown from "../../../commonComponents/Dropdown";
-import RadioGroup from "../../../commonComponents/RadioGroup";
+import Checkboxes from "../../../commonComponents/Checkboxes";
```

Comment:
The original code was using the radio button as checkboxes which seemed a bit unusual. The markup in USWDS is slightly different so it seemed useful to apply the right one.